PowerPointLabs/PowerPointLabs/Ribbon1.cs

```diff
@@ -141,14 +141,26 @@ public string GetCustomUI(string ribbonId)
         // Set the visibility of the Drawing Tools Format Tab
         public bool ToggleVisibleFormatTab(Office.IRibbonControl control)
         {
-            return !disableFormatTab;
+            return !DisableFormatTab;
         }
 
         // Toggles the boolean controlling the visibility of the Drawing Tools Format Tab
         public void ToggleVisibility(Office.IRibbonControl control, bool pressed)
         {
-            disableFormatTab = !disableFormatTab;
+            DisableFormatTab = pressed;
             _ribbon.InvalidateControlMso("TabDrawingToolsFormat");
+            _ribbon.InvalidateControl("VisibleFormatShapes");
+        }
+
+        public void InitialiseVisibilityCheckbox()
+        {
+            _ribbon.InvalidateControl("VisibleFormatShapes");
+        }
+        
+        // Sets the default starting status of the checkbox (whether checked or not)
+        public bool SetVisibility(Office.IRibbonControl control)

```

Comment:
I think it should be renamed to "GetPressed" or "GetPressedCheckbox" because it is not obvious that this is related to the checkbox.

Reply:
method GetPressed for the control actually sets whether the checkbox is checked, not whether the box has been pressed. Hence the return value for it actually sets the state of the checkbox, hence the keyword "Set in the method"